specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```
@@ -0,0 +1,2367 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3.1-preview.4",
+    "contact": {
+      "name": "Microsoft Cognitive Services",
+      "url": "https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+      "email": "mlapi@microsoft.com"
+    },
+    "title": "Text Analytics Client",
+    "description": "The Text Analytics API is a suite of natural language processing (NLP)  services built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis,  key phrase extraction and language detection. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>"
```

@ctufts we should update this as well. @assafi

```suggestion
    "description": "The Text Analytics API is a suite of natural language processing (NLP) services built with best-in-class Microsoft machine learning algorithms. The API can be used to analyze unstructured text using NLP tasks such as sentiment analysis, named entity recognition (NER) of general and personally identifiable information domains, key phrase extraction (KPE) and language detection. It can also be used to batch NER and KPE operations asynchronously to process documents with a single request. Furthermore, this API provides an asynchronous operation for analyzing unstructured clinical and biomedical text using NER, entity linking, relation extraction, and entity negation tasks. More documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>"
```

---

Do we need to be this specific in the description? 

If so we'd aslo need to mention EL (in the first change for the sync tasks), Opinion mining (since we are mentioning PII), PHI, etc. I'd assume if the user wants this additional information they would follow the link to the docs?